### DISCUSSION TITLE
Custom data directory for stalwart mails

### DISCUSSION THREAD

**Discussion by AiroPi** [nextcloud/all-in-one#4143] [Questions] [answered]
I just installed stalwart from the community containers and it works fine (except for some issues I have with outgoing mails but it's not my goal so I will deal with it atm)

I installed stalwart to backup my mails without the need of paying additional storage on my main mailbox.

But nextcloud-aio-stalwart save mails on a named volume, that is stored on the main disk. I have not a lot of space on the OS disk, is there a solution to save the mails on an external drive ?

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-store-the-filesinstallation-on-a-separate-drive
